examples/objective-c/hello_world/ViewController.m

```diff
@@ -1,4 +1,4 @@
-#import <Envoy/Envoy-Swift.h>
+#import <Envoy/Envoy.h>
```

Comment:
This header got renamed, and there is also now a modulemap in the framework so in theory we could also import with `@import Envoy;`. In theory we could attempt to keep the old name, but this seems nicer anyways

Reply:
Yep this is nicer 👌